app/scripts/controllers/transactions/index.js

```diff
@@ -348,7 +348,7 @@ export default class TransactionController extends EventEmitter {
       this._requestTransactionApproval(txMeta, {
         shouldShowRequest: false,
       }).catch((error) => {
```

Comment:
I guess this could theoretically happen if some code somewhere is explicitly doing a `throw` on an object that is `null` or `undefined`, but still a strange one.